salt/renderers/gpg.py

```diff
@@ -37,6 +37,55 @@
     increasing the system entropy. On virtualised Linux systems, this can often
     be achieved by installing the ``rng-tools`` package.
 
+Import keys to a master
+************************
+
+If the keys already exist and need to be imported to the salt master, run the
+following to import them.
+
+.. code-block:: bash
+
+    gpg --import salt-master.private.key
+    gpg --import salt-master.gpg
+
+If the salt master runs as normal user, become this user before importing the
+keys. The default target dir will be ``~/.gnupg``. This can be overridden by
+the ``--homedir`` option. The keys must be at least readable for the runuser.
+
+Adjust trust level of imported keys
+***********************************
+
+In some cases, importing existing keys may not be enough an the trust level of
+the key needs to be adjusted. This can be done by editing the key. The ``key_id``
```

Comment:
Is there a way that a person can detect this condition? Do we know what error(s) might be shown? That would be very helpful here, I think.

Reply:
Yes, this can be seen with `gpg --edit-key key_id`. I shall clarify that.

Reply:
Updated line in question. Should be fine imho.